tests/PerformanceTests/Microsoft.Identity.Web.Perf.Client/TestRunner.cs

```diff
@@ -129,59 +193,50 @@ public async Task Run()
                         exceptionsDuringRun.AppendLine($"Exception in TestRunner at {i} of {userEndIndex} - {userStartIndex}: {ex.Message}");
                         exceptionsDuringRun.AppendLine($"{ex}");
                     }
-
-                    Console.Title = $"[{userStartIndex} - {userEndIndex}] #: {i}, Loop: {loop}, " +
-                        $"Time: {(DateTime.Now - startOverall).TotalMinutes:0.00}, " +
-                        $"Req: {requestsCounter}, Cache: {tokenReturnedFromCache}: {fromCache}, " +
-                        $"AuthFail: {authRequestFailureCount}, Fail: {catchAllFailureCount}";
-
-                    if (Console.KeyAvailable)
-                    {
-                        ConsoleKeyInfo keyInfo = Console.ReadKey();
-                        if ((keyInfo.Modifiers == ConsoleModifiers.Control && (keyInfo.Key == ConsoleKey.X || keyInfo.Key == ConsoleKey.C)) || keyInfo.Key == ConsoleKey.Escape)
-                        {
-                            cancelProcessing = true;
-                            break;
-                        }
-                    }
                 }
 
-                UpdateConsoleProgress(startOverall, elapsedTime, requestsCounter, tokenReturnedFromCache, authRequestFailureCount, catchAllFailureCount);
+                UpdateConsoleProgress(startOverall, elapsedTime, requestsCounter, tokenReturnedFromCache, authRequestFailureCount, catchAllFailureCount, userStartIndex, userEndIndex, loop);
 
                 ScalableTokenCacheHelper.PersistCache();
 
-                if (DateTime.Now >= finishTime)
-                {
-                    cancelProcessing = true;
-                }
+                //File.AppendAllText(System.Reflection.Assembly.GetExecutingAssembly().Location + ".exceptions.log", exceptionsDuringRun.ToString());
```

Comment:
>//File.AppendAllText(System.Reflection.Assembly.GetExecutingAssembly().Location + ".exceptions.log", exceptionsDuringRun.ToString()); [](start = 16, length = 133)

This information is useful for debugging and understanding the failures happening. I think this should be persisted and returned to what ever starts the test process allowing them to get hold of this information (running locally this can easily be grabbed between runs, but it may be forgotten). Perhaps the file should have an added time stamp.... (and there should be protection about running out of diskspace writing logs)

Reply:
Yes, will add this in subsequent PR.